CONTRIBUTING.md

```diff
@@ -0,0 +1,171 @@
+  * All unit tests must be 100% passing before the pull requests will be approved and merged.
+
+
+## Getting Started
+
+If you’d like to work on a pull request and you’ve never submitted code before, follow these steps:
+
+1. Set up a local development environment. Please refer to
+the [manual](https://equinor.github.io/template-fastapi-react/) for instructions on how to build, test, and run the template locally.
+2. If you want to implement a breaking change or a change to the core, ensure there’s an issue that describes what you’re doing and the issue has been accepted. You can create a new issue or just indicate you’re working on an existing issue. Bug fixes, documentation changes, and other pull requests do not require an issue up-front.
+
+After that, you’re ready working with code.
+
+## Working with Code
+
+We love pull requests and the process of submitting a pull request is fairly straightforward and generally follows the same pattern each time.
+
+In general, we follow the ["fork-and-pull" Git workflow](https://github.com/susam/gitpr). 
+
+Here's a quick guide:
+
+1. Create your own fork of the repository 
+2. Clone the project to your machine
+3. To keep track of the original repository add another remote named upstream
+```shell
+git remote add upstream git@github.com:equinor/template-fastapi-react.git
+```
+4. Create a branch locally with a succinct but descriptive name. 
+```shell
+git checkout -b improve-feature
+```
+4. Make the changes in the created branch. 
+5. Add and run tests for your changes (we only take pull requests with passing tests).
+```shell
+docker-compose run --rm api pytest
+docker-compose run --rm web yarn test
+```
+6. Add the changed files 
+```shell
+git add path/to/filename
+```
+7. Commit your changes using
+   the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) formatting for our commit messages. 
+```shell
+git commit -m "conventional commit formatted message"
+```
+8. Before you send the pull request, be sure to rebase onto the upstream source. This ensures your code is running on the latest available code.
+```shell
+git fetch upstream
+git rebase upstream/main
+```
+9. Push to your fork.
+```shell
+git push origin improve-feature
+``` 
+9. Submit a pull request to the original repository (via Github interface). Please provide us with some explanation of why you made the changes you made. For new features make sure to explain a standard use case to us. 
+
+That's it... thank you for your contribution!
+
+After your pull request is merged, you can safely delete your branch. 
+
+## Your First Contribution
+
+You can start by looking through these beginner and help-wanted issues.
+
+## How to report a bug
+
+Report something that is broken or not working as
+intended [here](https://github.com/equinor/template-fastapi-react/issues/new?assignees=&labels=type%3A+%3Abug+bug&template=bug-report.md&title=).
+
+If you find a security vulnerability, do NOT open an issue.
```

Comment:
Possibly say what to do instead? E.g. "contact us via email etc.".

Reply:
Updated